src/main/resources/edu/cornell/kfs/module/ld/document/datadictionary/CuYearEndSalaryExpenseTransferDocument.xml

```diff
@@ -18,20 +18,47 @@
    - along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->
 
-  <bean id="YearEndSalaryExpenseTransferDocument" parent="YearEndSalaryExpenseTransferDocument-parentBean">
+  <bean id="CuYearEndSalaryExpenseTransferDocument" parent="YearEndSalaryExpenseTransferDocument-parentBean">
+    <property name="allowsErrorCorrection" value="true"/>
+    <property name="documentTypeName" value="YEST"/>
     <property name="documentClass" value="edu.cornell.kfs.module.ld.document.CuYearEndSalaryExpenseTransferDocument"/>
+    <property name="baseDocumentClass" value="edu.cornell.kfs.module.ld.document.CuYearEndSalaryExpenseTransferDocument"/>
+    <property name="businessRulesClass" value="org.kuali.kfs.module.ld.document.validation.impl.LaborDocumentRuleEngineRuleBase"/>
```

Comment:
Building on Jay's suggestion, unless we override the properties inherited from the parent bean, they don't need to be included. That would leverage the inheritance you now have here, make this file a little cleaner and help us avoid issues if the properties change in base code.